device/celestica/x86_64-cel_e1031-r0/sonic_platform/thermal.py

```diff
@@ -109,12 +109,16 @@ def __init__(self, thermal_index, airflow):
 
         self.name = self.get_name()
         self.postion = self._thermal_info["postion"]
-        self.minimum_thermal = self.get_temperature()
-        self.maximum_thermal = self.get_temperature()
+        self.minimum_thermal = None
+        self.maximum_thermal = None
 
     def _get_hwmon_path(self):
         hwmon_path = os.path.join(
             I2C_ADAPTER_PATH, self._thermal_info["i2c_path"])
+        if self._thermal_info["i2c_path"] == "i2c-11/11-001a/hwmon":
```

Comment:
@lizhijianrd exact path depends upon the hwmon enumerating the devices. We don't need this hard coding here. Ensuring the platform driver gets installed before reboot cause determination process runs is sufficient to fix the issue

Reply:
Thanks for pointing this out. Updated.